test/augmentation/test_augmentation_3d.py

```diff
@@ -759,3 +760,86 @@ def test_gradcheck(self, device):
         input = torch.rand((3, 3, 3)).to(device)  # 3 x 3 x 3
         input = utils.tensor_to_gradcheck_var(input)  # to var
         assert gradcheck(RandomRotation3D(degrees=(15.0, 15.0)), (input, ), raise_exception=True)
+
+
+class TestRandomEqualize3D:
+    def smoke_test(self, device, dtype):
```

Comment:
Our previous tests are incorrectly named here :). I will update the others soon. Sorry for this.

```suggestion
    def test_smoke(self, device, dtype):
```